### DISCUSSION TITLE
FormLister - data being passed back into the form

### DISCUSSION THREAD

**Discussion by user83743** [evolution-cms/evolution#1918] [Q&A]
Hi @Dmi3yy . would you mind sharing some of your experience here please 🙂 . i tried your formlister form examples and i tried https://docs.evo.im/en/04_extras/formlister/500_examples/002_elementy_upravleniya.html and for the "RADIO BUTTONS" and "TICK BOXES" neither form works - they will not **remember** the selected radio button or tickbox values, if there is an error in the form , for example, when the form user forgets to fill in a required field or forgets the recaptcha . The forms DO remember the values of textboxes but not radio buttons or tickboxes. do you know how to fix that? A formlister form with radio buttons and tickboxes is not very useful if it always forgets the values in the radio-button/tickbox selection blocks. thanks in advance.

i tried those formlister form examples on a few browsers and none of them have a working form example where the form will remember the radio button or tickbox data being passed back into the form. so it's not browser specific. all browsers are the same. all browsers pass back the values entered into textboxes but none pass back radio button data or tickbox data. any ideas? any advice would be much appreciated.

**Comment by Dmi3yy** [comment]
Try do like in this video: 
https://m.youtube.com/watch?v=Ilt0thLJPhc&list=PLKTHZGrYLdtoMegu0vG1Isek4_HsZPtZf&index=9&t=16s

**Comment by user83743** [comment]
:) i didn't see what I was to try to make the formlister form work

**Comment by user83743** [comment]
do you have any examples of formlister forms that have a radio button, checkbox or dropdown list where the form remembers that data if the form has to resubmit itself?

**Comment by modxuser** [comment]
@Dmi3yy

This issue has been resolved.

See the [tutorial here](https://github.com/modxuser/Evo-Extras-Tutorials/blob/main/FormLister/15.%20FormLister:%20Remember%20Inputs%20%2B%20Options%20On%20Form%20Re-submission.md)
